includes/class-micropub-endpoint.php

```diff
@@ -895,21 +891,7 @@ protected static function load_input() {
 		} elseif ( ! $content_type ||
 			   'application/x-www-form-urlencoded' === $content_type ||
 			   'multipart/form-data' === $content_type ) {
-			static::$input = array();
```

Comment:
this got lost in the move. it should be added back at the top of `form_to_json`, right?

Reply:
It should be $input=array() but yes